### PR TITLE
Adds radio muffs for muh poor ears

### LINKED ISSUE #964: Felinds ear go ow even with bowmans

### PR DESCRIPTION
**Pull request by Autisem**
## About The Pull Request

ALlows players to insert there headsets into ear muffs, giving them 2 ear protections

I forgot to PR this so sorry for the delay. Maby new sprites in the future
closes #964

## Why It's Good For The Game

Makes felind munation techies less dead weight. Choosing between getting stun locked due to loud noises or having no radio communaction during battle is pretty bad

## Changelog
:cl:
add: The cat beasts are evolving and have realised that they can put their radio headsets inside of their earmuffs.
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
The inserts should be put in an NSV13 files, and the interaction changes should be tagged with an //NSV13 comment.

**Comment by Autisem** [comment]
> The inserts should be put in an NSV13 files, and the interaction changes should be tagged with an //NSV13 comment.

Should be fixed now?

**Comment by Kmc2000** [comment]
Minor changelog edit for grammar - 11/11/2020

File: code/modules/clothing/ears/_ears.dm [hunk 1]
```diff
@@ -22,6 +22,18 @@
 	. = ..()
 	AddElement(/datum/element/earhealing)
 
+/obj/item/clothing/ears/earmuffs/attackby(obj/item/I, mob/living/user) //NSV13 comment
+	if(istype(I,/obj/item/radio/headset))
+		var/obj/item/radio/headset/radiomuff/newmuf = new /obj/item/radio/headset/radiomuff(get_turf(user.loc))
+		newmuf.keyslot = I
+		newmuf.recalculateChannels()
+		to_chat(user,"<span class='notice'>You stuff the [I] into the [src].</span>")
+		qdel(I)
```
Comment (francinum) [inline code/modules/clothing/ears/_ears.dm:31]: uh, If I'm reading this right, you're setting the keyslot to the radio and then deleting it???

Get the keyslots of the radio itself.

Comment (francinum) [reply code/modules/clothing/ears/_ears.dm:31]: you also need to copy over the command and use_command values for loudmode as well.

**Comment by ta3370** [comment]
Bowmans headsets exist, and I think the muni helm may be a second layer. Either way, not all races need to be able to do all jobs. If you're going to go a race vulnerable to sound even with protection, don't sit around the big boomy things.

I'm definitely for normal headsets being able to be put into earmuffs as a makeshift one-layer protection bowsman, but a bowsman/already insulated headset shouldn't fit in earmuffs as it's already surrounded with insulation.

**Comment by Autisem** [comment]
> Bowmans headsets exist, and I think the muni helm may be a second layer. Either way, not all races need to be able to do all jobs. If you're going to go a race vulnerable to sound even with protection, don't sit around the big boomy things. Races are meant to have unique mechanical differences, not just be cosmetics, and as such I don't think the double-layer aspect is a positive thing.
> 
> I'm definitely for normal headsets being able to be put into earmuffs as a makeshift one-layer protection similar to bowsman, but a bowsman/already insulated headset (such as the Munitions headset) shouldn't fit in earmuffs as it's already surrounded with insulation itself.
> Overall as a single-layer protection makeshift-bowsman could be good for muni-squads who have a different radio to keep track of but need ear protection, but it should not be better than a default munitions headset/bowsman
> +1 to basic concept
> -1 to reason and expansion of it. If you're going to play a race, play the race, not just another human that looks fancy, which Felinid basically already is if they ignore a hatred of loud sounds.

Felinds with bowmans and munation tech helmets can still get flash banged, people disagreed initialy when I propsed making felinds not have -2 bang protection/

File: code/modules/clothing/ears/_ears.dm (hunk 1 above)
Comment (Autisem) [reply code/modules/clothing/ears/_ears.dm:31]: When testing it did allow me to access the radio frequencies, iam not entirely sure how/why it works either

Comment (Autisem) [reply code/modules/clothing/ears/_ears.dm:33]: Should be fixed now? I tested loudmode and it worked. It should also copy the key over now

**Comment by Kmc2000** [comment]
Testmerged

**Comment by Karmic-Skink** [comment]
Eh, tbh work arounds for species weaknesses makes the species having weaknesses kinda moot in the first place. Why not just make everything a human?

**Comment by MacBlaze1** [comment]
dont merge this PR it makes felinids weakness useless and snowflake purely

**Pull request closed without merging**